resources/sass/people.scss

```diff
@@ -249,17 +249,17 @@
   .profile {
     .sidebar-box {
       background-color: #f2f4f8;
-      border: 1px solid #eee;
+      border: 1px solid #eeeeee;
       border-radius: 3px;
-      color: #333;
+      color: #313436;
       margin-bottom: 25px;
       padding: 10px;
       position: relative;
       font-size: 0.875rem;
 
       &.edit {
-        background-color: #fffceb;
-        border-color: gold;
+        background-color: #fffacd;
+        border-color: #daa520;
```

Comment:
These colors are near, but the final aspect is quite different. Could we find closer colors as the originals?

Reply:
I’ve restored these two to their originals.